bigframes/operations/_matplotlib/core.py

```diff
@@ -74,18 +75,24 @@ def _compute_plot_data(self):
         return self._compute_sample_data(self.data)
 
 
-class LinePlot(SamplingPlot):
-    @property
-    def _kind(self) -> typing.Literal["line"]:
-        return "line"
-
-
 class AreaPlot(SamplingPlot):
     @property
     def _kind(self) -> typing.Literal["area"]:
         return "area"
 
 
+class BarPlot(SamplingPlot):
+    @property
+    def _kind(self) -> typing.Literal["bar"]:
+        return "bar"
```

Comment:
Does bar plot make sense as a sampling plot? It seems that bar plots are really meant for small-cardinalities and don't down-sample well.

Reply:
The bar plots are discrete while the area/line/scatter plots are continuous. Hence, the downsampling in the bar is not straightforward compared to others. Thanks for suggestions here. I just added a warning message to suggest users to consider pre-processing data (aggregations or select top categories).